sdk/keyvault/azure-security-keyvault-keys/test/ut/key_cryptographic_client_test_live.cpp

```diff
@@ -276,3 +276,16 @@ TEST_F(KeyVaultClientTest, RemoteSignVerifyDataRSA256)
     EXPECT_TRUE(verifyResult.IsValid);
   }
 }
+
+namespace {
+static std::string GetSuffix(const testing::TestParamInfo<int>& info)
+{
+  return std::to_string(info.param);
+}
+} // namespace
+
+INSTANTIATE_TEST_SUITE_P(
+    Parametrized,
+    KeyVaultClientTest,
+    ::testing::Values(13, 55, 233, 987, 1597, 2048),
```

Comment:
Are all these key sizes valid? Is 2048 the max? What about negatives, 0, or larger than 2048?

Reply:
in the context of the C++ api please define valid, there are some fibonacci sequence numbers, i also added some negatives, and some higher powers of 2. 
When we call create key with the key length provided here, we end up calling an azure API  with the parameters we want (one of them being the key size). 
We receive a response of 200and a key back. 
All we have to go on here for defining valid is "will i be able to pass the value to Azure". i don't think we should care beyond this point, what azure does with the key, we should not start testing Azure keyvault implementation. Since Azure graciously comes back with a key, then i would say it's "valid"